features/performance/test/kubeletconfig_test.go

```diff
@@ -0,0 +1,67 @@
+package performance_test
+
+import (
+	"bytes"
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
+	kubeletconfigv1beta1 "k8s.io/kubelet/config/v1beta1"
+
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	mcfgScheme "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned/scheme"
+)
+
+const kubeletconfigYaml = "12-kubeletconfig-worker-rt.yaml" // TODO pass it as a param?
+
+var _ = Describe("TestPerformanceKubeletConfig", func() {
+	var _ = Context("CPU Manager policy", func() {
+		It("Should set the policy to 'static'", func() {
+			// parameters not really relevant for this test, just use something valid
+			kc := loadKubeletConfig(kubeletconfigYaml, "1-15", "0")
+			Expect(kc).ToNot(BeNil())
+			specKubeletConfig, err := decodeKubeletConfig(kc.Spec.KubeletConfig.Raw)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(specKubeletConfig.CPUManagerPolicy).To(Equal("static"))
+		})
+	})
+
+	// see: https://github.com/openshift-kni/baremetal-deploy/pull/56
```

Comment:
Can we remove the comment here? That PR is merged, and IMHO it's not important anymore to know about it

Reply:
Done